src/xml/merge-multi-xmi.xsl

```diff
@@ -16,6 +16,9 @@
     
     <xsl:import href="../../config/config-proxy.xsl"/>
     
+    <!-- List of xmi files to be merged into a single xmi file-->
+    <xsl:variable name="xmiFileList" select="collection('../../test/test-multi-xmi?select=*.xml')"/>
```

Comment:
why do we want to take XMI files from a "test" location? 
This shall not be in the master branch. We need a proper mechanism to indicate where is the input location.